.circleci/config.yml

```diff
@@ -169,6 +169,23 @@ workflows:
       - launch:
          name: "wandb launch tests"
          toxenv: "pylaunch,covercircle"
+      #
+      # sharded unittests
+      #
+      - test:
+         name: "unit-s_nb-lin-py36"
+         image: "python:3.6"
```

Comment:
Why (only) 3.6?

Reply:
For now, i figured by testing the oldest supported version we would find the most issues.  If we test everything then i think the amount of test resources we are billed will get a bit overwhelming... notice the functional tests also only test one python version.